src/back/controllers/api/routes/my/my.js

```diff
@@ -14,5 +14,6 @@ function responseFactory(action) {
 module.exports = {
```

Comment:
It is better to give opportunity for getting all necessary arguments from request to action. challengeId - is not enough for actions.